app/views/tag/show.html.erb

```diff
@@ -4,7 +4,7 @@
   <%= render partial: 'tag/show/header' %>
   
   <% if @tags.first && params[:id][-1..-1] != "*" %>
-    <%= render :partial => "tag/graph", locals: { tags: @tags } %>
+    <%= render partial : "tag/graph", locals: { tags: @tags } %>
```

Comment:
Thanks for making a pull request, you're doing great! I believe that the errors in your travis are caused by the syntax of this line. You should remove the space before the colon because it's not proper ruby partial syntax.
```suggestion
    <%= render partial: "tag/graph", locals: { tags: @tags } %>
```

In the future, you should explain why you changed a particular line of code if it's not directly related to the topic of your commit message. Let me know if it helps with the error :)